angular1/app/components/version/interpolate-filter.ts

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('myApp.version.interpolate-filter', [])
+
+.filter('interpolate', ['version', function(version) {
```

Comment:
I think it'd be great if this sample worked under `--noImplicitAny`. Maybe in a future PR